client/wfnews-war/src/main/angular/src/app/components/dashboard-component/widgets/historical-comparison-widget/historical-comparison-widget.component.html

```diff
@@ -1,7 +1,32 @@
 <div class="container">
   <div class="widget-header">
-    <span>Historical Comparison</span>
+    <span class="widget-title">Historical Comparison</span>
+    <div class="widget-button-bar">
+      <button [disabled]="!viewWildfireCounts" (click)="viewWildfireCounts = !viewWildfireCounts" *ngIf="!viewWildfireCounts" class="button-selected" mat-flat-button ><img height="24" width="24" src="/assets/images/svg-icons/measure-blue.svg" alt="Hectares" class="button-icon" /><span class="button-label-selected">Hectares Burned</span></button>
+      <button (click)="viewWildfireCounts = !viewWildfireCounts" *ngIf="viewWildfireCounts" class="button" mat-flat-button ><img height="24" width="24" src="/assets/images/svg-icons/measure-black.svg" alt="Hectares" class="button-icon" /><span class="button-label">Hectares Burned</span></button>
+      <button (click)="viewWildfireCounts = !viewWildfireCounts" *ngIf="!viewWildfireCounts" class="button" mat-flat-button ><img height="24" width="24" src="/assets/images/svg-icons/wildfire-black.svg" alt="Wildfires" class="button-icon" /><span class="button-label">Wildfires</span></button>
+      <button [disabled]="viewWildfireCounts" (click)="viewWildfireCounts = !viewWildfireCounts" *ngIf="viewWildfireCounts" class="button-selected" mat-flat-button ><img height="24" width="24" src="/assets/images/svg-icons/wildfire-blue.svg" alt="Wildfires" class="button-icon" /><span class="button-label-selected">Wildfires</span></button>
+    </div>
   </div>
   <ng-template #loading><mat-spinner [diameter]="80" class="spinner-position"></mat-spinner></ng-template>
-  <div *ngIf="startupComplete; else loading"></div>
+  <div class="content" *ngIf="startupComplete; else loading">
+    <div class="content-title">{{viewWildfireCounts ? 'Total Wildfires' : 'Hectares Burned Each Year'}}</div>
+    <div class="content-message">{{viewWildfireCounts ? 'One hectare is a unit of area equal to 10,000 square meters or about 2.5 acres of land.' : 'One hectare is a unit of area equal to 10,000 square meters or about 2.5 acres of land.'}}</div>
```

Comment:
Note, I am aware the two strings are identical, but that's what's in the mockup. I highly suspect the mockup is incorrect and one of these strings will change to be more Fire Totals specific vs. hectare definitions